### DISCUSSION TITLE
Show list of types that implement a given interface

### DISCUSSION THREAD

**Discussion by pepone** [dotnet/docfx#8914] [General]
In Microsoft APIs reference, you can see the types that implement an interface, for example:
<img width="579" alt="derived" src="https://github.com/dotnet/docfx/assets/254604/2a4dde78-2e27-4126-a097-0447306c02e3">

Is this doable with docfx?

**Comment by yufeih** [comment]
Seems like docfx currently shows derived types for classes but not for interfaces.

<img width="342" alt="image" src="https://github.com/dotnet/docfx/assets/511355/26d9d365-985d-473d-abc8-099d5ab0dabc">

**Reply by pepone** [reply]
Interesting, any idea why it is not shown for interfaces?
